nautobot/core/templates/generic/object_detail.html

```diff
@@ -142,6 +142,7 @@ <h1>{% block title %}{{ object }}{% endblock %}</h1>
                 </div>
             </div>
         </div>
+	{% block extra_tab_content %}{% endblock extra_tab_content %}
```

Comment:
This is new and should be documented as part of the release notes. I had to do it out of necessity to be able to allow a detail view to add extra tab content on the main tab.